akvo/rsr/views/my_rsr.py

```diff
@@ -339,20 +327,13 @@ def project_editor(request, project_id):
         # Default indicator
         'default_indicator': default_indicator,
 
-        # Custom fields
-        'custom_fields_section_1': custom_fields_section_1,
-        'custom_fields_section_2': custom_fields_section_2,
-        'custom_fields_section_3': custom_fields_section_3,
-        'custom_fields_section_4': custom_fields_section_4,
-        'custom_fields_section_5': custom_fields_section_5,
-        'custom_fields_section_6': custom_fields_section_6,
-        'custom_fields_section_7': custom_fields_section_7,
-        'custom_fields_section_8': custom_fields_section_8,
-        'custom_fields_section_9': custom_fields_section_9,
-        'custom_fields_section_10': custom_fields_section_10,
-        'custom_fields_section_11': custom_fields_section_11,
     }
 
+    # Custom fields context
+    for section_id in xrange(1, 12):
```

Comment:
Made a minor change here to directly generate `section_id`, instead of doing `i + 1`.